OTAnalytics/plugin_ui/nicegui_gui/pages/visualization_filters_form/container.py

```diff
@@ -0,0 +1,219 @@
+from nicegui import ui
+
+from OTAnalytics.adapter_ui.abstract_frame_filter import AbstractFrameFilter
+from OTAnalytics.adapter_ui.default_values import DATETIME_FORMAT
+from OTAnalytics.adapter_ui.dto import DateRangeDto
+from OTAnalytics.adapter_ui.view_model import ViewModel
+from OTAnalytics.application.logger import logger
+from OTAnalytics.application.resources.resource_manager import (
+    ProjectKeys,
+    ResourceManager,
+    VisualizationFiltersKeys,
+)
+from OTAnalytics.domain.date import DateRange
+from OTAnalytics.plugin_ui.customtkinter_gui.frame_filter import (
+    InvalidDatetimeFormatError,
+)
+from OTAnalytics.plugin_ui.nicegui_gui.nicegui.elements.forms import DateTimeForm
+from OTAnalytics.plugin_ui.nicegui_gui.pages.configuration_bar.svz_metadata_form import (  # noqa
+    SvzMetadataForm,
+)
+
+
+class VisualizationFiltersForm(AbstractFrameFilter):
+
+    def __init__(
+        self,
+        resource_manager: ResourceManager,
+        view_model: ViewModel,
+    ) -> None:
+        self._resource_manager = resource_manager
+        self._viewmodel = view_model
+        self._filter_by_date: ui.checkbox
+        self._filter_by_class: ui.checkbox
+        self._filter_by_class_list: list = []
+        self._introduce_to_viewmodel()
+        self._frames_to_skip = 0
+        self._seconds_to_skip = 0
+
+        self._start_date = DateTimeForm(
+            self._resource_manager.get(ProjectKeys.LABEL_START_DATE),
```

Comment:
Please add keys in `VisualizationFiltersKeys` for `start_date`, `end_date`, `start_time` and `end_time`. The texts are currently the same but may differ in the future.